static/src/stylesheets/module/commercial/glabs/_hosted.scss

```diff
@@ -1,13 +1,17 @@
 $hosted-stripe: 28px;
+$hosted-video-width: 960px;
+$hosted-video-height: 540px;
 
 @mixin hosted-fade-in {
+    visibility: visible;
     opacity: 1;
-    transition: opacity 1s linear;
+    transition: opacity 1s linear, visibility 0s;
```

Comment:
Do we actually want the things we're making invisible to be hidden from screen readers, and if we do, should we flag them with ARIA live regions?